CMakeLists.txt

```diff
@@ -191,13 +191,19 @@ message(STATUS "CMAKE_CXX_COMPILER_ID = ${CMAKE_CXX_COMPILER_ID}")
 if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
   if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
       if(${CMAKE_CXX_COMPILER_ID} MATCHES Clang)
+        if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.0)
+          message( FATAL_ERROR "Please use Version 12.0 or greater")
+        endif(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 12.0)
```

Comment:
how about moving this check outside this anyway super nasty section?

Then you also dont need to duplicate it

Reply:
yep you are right....